### PR TITLE
Bring K8s Spark to near parity with the DC/OS spark demo

### PR DESCRIPTION
**Pull request by SorraTheOrc**

### CODE REVIEW

**Comment by SorraTheOrc** [comment]
@suhaildawood can you review this PR please

File: incubator/kubernetes_spark_install/preparation/script.md [hunk 1]
```diff
@@ -174,7 +171,8 @@ The Kubernetes [GitHub repository](https://github.com/kubernetes/kubernetes) inc
 Let's clone the repository so that we can install Spark and Zeppelin easier later on.
 
 ```
-git clone https://github.com/kubernetes/kubernetes.git ~/kubernetes
+mkdir -p ~/.acs/demo
+git clone https://github.com/kubernetes/kubernetes.git ~/.acs/kubernetes
```
Comment (suhaildawood) [inline incubator/kubernetes_spark_install/preparation/script.md]: Looks like this needs to be changed to `~/.acs/demo/kubernetes`

File: incubator/kubernetes_spark_install/script.md [hunk 2]
```diff
@@ -22,14 +27,14 @@ your application configuration. You select the size, number of hosts,
...
 else.
 
-# Deploy Spark
-
-It is assumed that you have prepared the demo environment by running `prep.sh`, if not you need to break from this script and run it now.
+# Deploy Spark and Zeppelin
 
-We can create a new Spark cluster through the Kubernetes cli and the `yml` files found in the `~/kubernetes/examples/spark` directory. We will also spin up Zeppelin.
+We can create a new Spark cluster through the Kubernetes cli and the
+`yml` files found in the `~/.acs/kubernetes/examples/spark` directory
```
Comment (suhaildawood) [inline incubator/kubernetes_spark_install/script.md]: `~/.acs/demo/kubernetes/examples/spark`

File: incubator/kubernetes_spark_install/script.md [hunk 3]
```diff
@@ -22,14 +27,14 @@ your application configuration. You select the size, number of hosts,
...
+(these were checked out for you by the preparation script). 
 
 ```
-kubectl create -f ~/kubernetes/examples/spark
+kubectl create -f ~/.acs/kubernetes/examples/spark
```
Comment (suhaildawood) [inline incubator/kubernetes_spark_install/script.md]: `~/.acs/demo/kubernetes/examples/spark`

**Review by suhaildawood — CHANGES_REQUESTED** [review]
Looks good, the only thing I see is the kubernetes repo pointing to different folders.

**Pull request merged**